geoips/plugins/modules/procflows/config_based.py

```diff
@@ -770,6 +770,11 @@ def call(fnames, command_line_args=None):
     bg_self_register_dataset = None
     bg_self_register_source = None
 
+    if command_line_args.get("no_sectoring") is not None:
```

Comment:
Probably add the new checklist to the PR, and make sure everything is tested.  Perhaps we want to add a unit test that actually uses the --no-sectoring option also?

Reply:
@mindyls Adding a new test for this would also require adding a new output file to test against, right? I don't really know how to do that. Would you be able to add a test script that uses `--no-sectoring` and add the test file?

I think we should document how to add a new tests script with its test file somewhere in the developer docs.

Reply:
Adding test scrxipts / outputs is part of template_basic_plugin currently.  There is a lot that is part of template_basic_plugin

Reply:
@jsolbrig I modified the GMI test script to use the --no_sectoring option.  note this forced the output filenames to change (since the start time in the filename is determined by the sectored data), so I updated the test outputs as well.